lib/handlerUtils.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-unused-vars */
+const { decode, verify } = require('jsonwebtoken');
+const { promisify } = require('util');
+const config = require('./config');
+const logger = require('./logger');
+const findUsersByEmail = require('./findUsersByEmail');
+
+const validateAuth0Token = async (childtoken) => {
+  try {
+    const decoded = decode(childtoken, { complete: true });
+    const jwtVerifyAsync = promisify(verify);
+    const key = config('AUTH0_CLIENT_SECRET');
+    if (!key) {
+      return false;
+    }
+
+    const verifyOptions = {
+      audience: config('AUTH0_CLIENT_ID'),
+      issuer: `https://${config('AUTH0_DOMAIN')}/`,
+      algorithms: ['HS256']
+    };
+    await jwtVerifyAsync(childtoken, key, verifyOptions);
+
+    return decoded.payload;
```

Comment:
We should be able to use the output of `verify`, I don't think there is a need to decode anymore

https://github.com/auth0/node-jsonwebtoken/blob/bc28861f1fa981ed9c009e29c044a19760a0b128/verify.js#L251-L261

Reply:
Sounds good I'll remove decoding

Reply:
Can't update that until we get the update to `jsonwebtoken` from webtask